tests/size_test.py

```diff
@@ -194,6 +196,10 @@ def testScientificNotation(self):
         self.assertEqual(size.parseSpec("1E-4KB"), Decimal("0.1"))
         self.assertEqual(Size("1E-10KB"), Size(0))
 
+    def testPickling(self):
+        s = Size("10 MiB")
+        self.assertEqual(s, cPickle.loads(cPickle.dumps(s)))
```

Comment:
When we're unit-testing size...we are testing that size's methods work correctly.
```__reduce__``` is used for other things than pickling, like implementing copying.

I agree that it is important that the pickling works...but there is some better place to test that.
blivet-gui is the app that needs the ```__reduce__``` method for pickling...it should have tests to make sure that the pickling is successful for its purposes. At some point, blivet-gui should have _some_ tests, perhaps this is a good place to begin.